src/main/java/com/acrolinx/sidebar/utils/XMLLookupUtils.java

```diff
@@ -187,15 +192,23 @@ private static String getDocumentXml(Document document) throws IOException {
 
   private static Document buildDocument(String xmlContent)
```

Comment:
for consistency, we should rename this method to "buildDocumentWithNamespaceAwareness" I think?